tests/unit/adaptors/test_importscanner.py

```diff
@@ -927,6 +927,67 @@ def test_get_raw_imports(self):
             ),
         }
 
+    @pytest.mark.xfail(strict=True)
+    def test_multiline_comment_a(self):
```

Comment:
https://github.com/Peter554/pyimportparse/commit/e7d3d0da843ce3ea36d5a8342d3cb1bbdb32ea05 might be an option

Reply:
On reflection, I think this might be a better approach -> https://github.com/seddonym/grimp/pull/214